lib/charms/grafana_k8s/v0/grafana_dashboard.py

```diff
@@ -947,9 +955,11 @@ def _replace_uid(
         # already rendered there, so we do not want to overwrite it with a uid generated from aggregator's info.
         # We overwrite the uid only if it's not a valid "Path40" uid.
         if not DashboardPath40UID.is_valid(original_uid := dashboard_dict.get("uid", "")):
+            # Replace _is_subpath with dashboard_path.is_relative_to(charm_dir) when minimum Python is set to 3.9
```

Comment:
```suggestion
            # TODO: Replace _is_subpath with dashboard_path.is_relative_to(charm_dir) when minimum Python is set to 3.9
```

Could you also open an issue for this and link it in a comment link below the TODO?